lib/widgets/lightbox.dart

```diff
@@ -89,6 +93,157 @@ class _CopyLinkButton extends StatelessWidget {
   }
 }
 
+// class _DownloadImageButton extends StatelessWidget {
```

Comment:
We don't merge large, unexplained blocks of commented-out code; please remove all this.

Reply:
That was a mistake, have removed it now,